packages/babel-cli/src/babel/util.js

```diff
@@ -50,12 +50,12 @@ export function log(msg) {
   if (!commander.quiet) console.log(msg);
 }
 
-export function transform(filename, code, opts) {
+export function transform(filename, code, opts, cb) {
   opts = Object.assign({}, opts, {
     filename,
   });
 
-  return babel.transform(code, opts);
+  babel.transform(code, opts, cb);
 }
 
 export function compile(filename, opts) {
```

Comment:
Sorry, I didn't realize this function used a different version, but we'll want to update both. This function uses `babel.transformFileSync` and ideally this same PR should also update that to use the async `babel.transformFile`.

That will probably be the complicated part of this task because the code that uses `util.compile` is mostly synchronous right now.